### PR TITLE
Feature#28 crud places

### PR DESCRIPTION
**Pull request by fego-dev**
# Description
What did you implemented/Why did you implemented this:
 - I created a new module for Place.
 - The place is a very important module because it is used by the user to register their stay in a place.
 - Place has a method for generate QR CODE.
 - Place module is for api and web app.
 
  ## Screenshots 
List / Delete / Search
imagen
Create / Edit
imagen

### CODE REVIEW

**Review by LPMendoza — APPROVED** [review]
You did a very good work, I think all is ok

File: app/assets/stylesheets/places.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Places controller here.
```
Comment (ccarruitero) [inline app/assets/stylesheets/places.scss:1]: Avoid unused files

File: app/controllers/api/v1/application_api_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Aplication controller for api
+class API::V1::ApplicationAPIController < ActionController::Base
```
Comment (ccarruitero) [inline app/controllers/api/v1/application_api_controller.rb:2]: Nice :ok_hand:

File: app/helpers/places_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PlacesHelper
```
Comment (ccarruitero) [inline app/helpers/places_helper.rb:1]: unused file

File: test/models/place_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (ccarruitero) [inline test/models/place_test.rb:1]: unused file

**Review by ccarruitero — COMMENTED** [review]
Seems good, just a few comments

Also, will be great if you add tests for you api's controllers

**Review by ArielMD — APPROVED** [review]
Good work, that's fine with me

File: app/controllers/places_controller.rb [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      if @place.save
+        respond_if_is_true_web(format, places_url, 'Place was successfully created.', :show, :created, @place)
+      else
+        respond_if_is_false_web(format, :new, :unprocessable_entity, @place.errors, :unprocessable_entity)
+      end
+    end
+  end
+
+  # PATCH/PUT /places/1 or /places/1.json
+  def update
```
Comment (MiguelOcegueraM) [inline app/controllers/places_controller.rb:43]: I like this!

**Review by MiguelOcegueraM — APPROVED** [review]
Everything seems good, your code is very clean but in you have some things that your aren't using so try to delete and delete the commented code.

**Pull request merged**